From 98a66f345768562d438af6dd6ddc418eca227fbc Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Fri, 8 Jan 2016 17:51:38 +0200
Subject: [PATCH 77/86] arch/arm: Fix Compiler Warnings

The Axxia updates used NR_CPUS instead of CONFIG_NR_CPUS.

Signed-off-by: John Jacques <john.jacques@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm/include/asm/kmap_types.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/include/asm/kmap_types.h b/arch/arm/include/asm/kmap_types.h
index 6ee0cf8..c7877de 100644
--- a/arch/arm/include/asm/kmap_types.h
+++ b/arch/arm/include/asm/kmap_types.h
@@ -6,7 +6,7 @@
 /*
  * This is the "bare minimum".  AIO seems to require this.
  */
-#if (NR_CPUS > 15)
+#if (CONFIG_NR_CPUS > 15)
 /* Prevent overlap between fixmap mapping and CPU vector page for 16th core */
 #define KM_TYPE_NR 15
 #else
-- 
1.7.5.4

